tests/wallet/sync/test_wallet_sync.py

```diff
@@ -58,6 +153,18 @@ async def test_basic_sync_wallet(self, bt, two_wallet_nodes, default_400_blocks,
                 100, wallet_height_at_least, True, wallet_node, len(default_400_blocks) + num_blocks - 5 - 1
             )
 
+    @pytest.mark.parametrize(
+        "two_wallet_nodes",
+        [
+            dict(
+                disable_capabilities=[Capability.BLOCK_HEADERS],
+            ),
+            dict(
+                disable_capabilities=[Capability.BASE],
+            ),
+        ],
+        indirect=True,
+    )
```

Comment:
I would have expected this parameterization to be applied to the `wallet_node` or possibly `two_wallet_nodes` fixtures. That way all tests would be run against both sets of capabilities.

Reply:
whatever level those fixtures are constructed at. my point is that if those fixtures are parameterized, all tests using them will also be parameterized, which means we cover both new and old clients with the existing tests

Reply:
I think it's overkill to run all the tests again for this change. however I do see an argument  for doing it for the wallet sync test, i'll do that